### PR TITLE
Bug 1798077: Cluster Inventory card shows both VM + VMI count

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/inventory.tsx [hunk 1]
```diff
@@ -81,15 +81,27 @@ export const getVMStatusGroups: StatusGroupMapper = (
...
+  const vmStatuses = vms.map((vm: VMKind) => {
     const vmi = (vmis || []).find((instance) => getName(vm) === getName(instance));
-    const { status } = getVMStatus({ vm, vmi, pods, migrations });
+    return getVMStatus({ vm, vmi, pods, migrations }).status;
+  });
+
+  const vmisWithoutVM = (vmis || []).filter(
+    (instance) => !vms.find((vm) => getName(vm) === getName(instance)),
+  );
+  const vmiStatuses = vmisWithoutVM.map((vmi: VMIKind) => {
```
Comment (rawagner) [inline frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/inventory.tsx]: just a nit - no need to specify `VMIKind` type and can be one-liner
```suggestion
  const vmiStatuses = vmisWithoutVM.map((vmi) => getVMStatus({ vm: undefined, vmi, pods, migrations }).status);
```

Comment (mareklibra) [reply frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/inventory.tsx]: done

**Pull request merged**